.github/workflows/gh_actions_push.yaml

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2012-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+name: Nightly Dockerimage Build
+
+on:
+  push:
+    branches: [ master ]
+jobs:
+  build-push:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout che-machine-exec source code
+      uses: actions/checkout@v2
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+    - name: Login to quay.io
+      uses: docker/login-action@v1 
+      with:
+        username: ${{ secrets.QUAY_USERNAME }}
+        password: ${{ secrets.QUAY_PASSWORD }}
+        registry: quay.io
+    - name: Build and push nightly
+      uses: docker/build-push-action@v2
+      with:
+        file: build/dockerfiles/Dockerfile
+        platforms: linux/amd64,linux/ppc64le,linux/s390x
+        push: true
+        tags: quay.io/eclipse/che-machine-exec:nightly
```

Comment:
Do we want to use nightly tag for each commit build? 
We used to use `:next` tag for such purposes

Reply:
Well, Right now is used nightly tag in every commit

Reply:
Also build the image with commit sha.

Reply:
> Right now is used nightly tag after merging every PR. It is not for each commit build

If I understand you correctly - then yes, technically if you merge PR with multi commits - then new `nightly` is rebuilt once. But it does not make a lot of difference

If you take a look how nightly was built previously - it's built every night at 12 AM https://ci.centos.org/job/devtools-che-machine-exec-nightly/
built the latest image after PR is merged should work fine but I just wanted to bring that it may be confusing to still use nightly tag... `next` should be better descriptive.

Reply:
We use nightly tag everwhere https://github.com/eclipse/che-plugin-registry/blob/master/v3/plugins/eclipse/che-machine-exec-plugin/nightly/meta.yaml#L27
we don't use next tag for machine exec plugin.
@sleshchenko 
Do you think it can be another task?

Reply:
I believe `Switching to next tag` can be another task... But I prefer not to push next changes to nightly tag... So, then please consider changing
```
on:
  push:
    branches: [ master ]
```
to something like - 
```
on:
  schedule:
    # * is a special character in YAML so you have to quote this string
    - cron:  '0 0 * * *'
```

Reply:
It makes sense for me to switch to cron job as well

Reply:
Done.